Makefile

```diff
@@ -78,11 +78,10 @@ STDLIB_TESTS := \
   re_tests \
   sys_test \
   tempfile_test \
-  test/seq_tests \
```

Comment:
They exported classes for actual tests file like test_tuple, test_list, but they dont do actual test themselves, take a look at one of those.